src/platforms/ruby/common/configuration/options.mdx

```diff
@@ -82,18 +82,22 @@ end
 
 `breadcrumbs_logger`
 
-: Sentry supports two breadcrumbs loggers in the Ruby SDK:
-- `SentryLogger` - A general breadcrumbs logger for all Ruby applications.
-- `ActiveSupportLogger` - Built on top of Rails ActiveSupport's instrumentation and provides many Rails-specific information.
+: Sentry supports different breadcrumbs loggers in the Ruby SDK:
+- `:sentry_logger` - A general breadcrumbs logger for all Ruby applications.
+- `:active_support_logger` - Built on top of Rails ActiveSupport's instrumentation and provides many Rails-specific information.
+- `:http_logger` - It captures requests made with the standard `net/http` library.
```

Comment:
the `:` is displaying for these. Do we mean it to? 
![Screen Shot 2021-05-03 at 1 59 49 PM](https://user-images.githubusercontent.com/61481573/116933007-d56f5280-ac17-11eb-8b3e-a93bbb367998.png)

Reply:
Yes, that means Ruby's `symbol` type.